reconstruction/rtpc/src/main/java/org/jlab/rec/rtpc/hit/KalmanFilter.java

```diff
@@ -0,0 +1,2417 @@
+package org.jlab.rec.rtpc.hit;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.Writer;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+import org.ejml.simple.SimpleMatrix;
+import org.jlab.clas.swimtools.Swim;
+import org.jlab.io.base.DataBank;
+import org.jlab.io.base.DataEvent;
+
+
+public class KalmanFilter {
```

Comment:
This seems to be the old version: should this stay? even if it would be deleted, it could always be retrieved from git